jenkins-builds/production.sh

```diff
@@ -96,17 +96,15 @@ if [[ "$system" =~ "daint" || "$system" =~ "dom" ]]; then
         echo -e "\n No architecture defined. Please use the option -a,--arch to define the architecture \n"
         usage
     else
-        module rm ddt
-        module rm xalt
-        module rm PrgEnv-cray
-        module use /opt/cray/pe/craype/2.5.8/modulefiles
+        module purge
+        module load craype craype-network-aries modules
         module load daint-${ARCH}
         eb_args="${eb_args} --modules-header=${scriptdir%/*}/login/daint-${ARCH}.h"
     fi
 fi
 
 # --- COMMON SETUP ---
-export EB_CUSTOM_REPOSITORY=$PWD/easybuild
+export EB_CUSTOM_REPOSITORY=/apps/common/UES/jenkins/production/easybuild
```

Comment:
Why not wrap the variable definition with a `if` statement. So that if it is defined, one can use the defined one, if not defined, one could define this default?
Some think like:
```bash
if [ -z $EB_CUSTOM_REPOSITORY ]; then
    export EB_CUSTOM_REPOSITORY=/apps/common/UES/jenkins/production/easybuild
fi
```